Cargo.toml

```diff
@@ -22,7 +22,7 @@ blt-backend = ["cursive/blt-backend"]
 [dependencies]
 syn = "=1.0.57"
 serde_json = "1.0.61"
-log = "0.4.13"
+log = { version = "0.4.13", features = ["release_max_level_off"]}
```

Comment:
Here we should remove the "release_max_level_off" feature because this disables logging completely (we need that for crash reports)